Resources/Prototypes/Entities/Mobs/Species/arachnid.yml

```diff
@@ -30,6 +30,9 @@
     spawned:
       - id: FoodMeatSpider
         amount: 5
+  - type: Inventory
+    templateId: arachnid
+    speciesId: arachnid # DeltaV - allow arachnid clothing sprite variants
```

Comment:
add the speciesId bit to the existing Inventory component at the bottom (dont remove it just to place it here)